applications/MeshingApplication/tests/test_mpi_parmmg.py

```diff
@@ -14,7 +14,7 @@ def GetFilePath(fileName):
 
 class TestMPIParMmg(KratosUnittest.TestCase):
 
-    @KratosUnittest.skipUnless(IsDistributedRun() and ParallelEnvironment.GetDefaultSize() == 2, "Test designed to be run with two ranks.")
+    @KratosUnittest.skipUnless(IsDistributedRun() and ParallelEnvironment.GetDefaultSize() <= 4,  "Test designed to be run with max. 4 ranks.")
```

Comment:
so the results are actually different when using dfferent number of ranks?

Reply:
Yes, they are different.

ParMmg tries to generate a new metric by remeshing the partitions independently. Then it changes the interfaces on the mesh and repeats the process. I guess this causes different world sizes to give different results (since interfaces are different). I tried increasing the number_of_iterations on ParMmg but still did not get exactly the same mesh afterwards.

Reply:
ah ok I didn't get that, sorry for the confusion

in this case I guess the solution you have now is ok